VoodooI2CHID/VoodooI2CPrecisionTouchpadHIDEventDriver.cpp

```diff
@@ -12,8 +12,25 @@
 OSDefineMetaClassAndStructors(VoodooI2CPrecisionTouchpadHIDEventDriver, VoodooI2CMultitouchHIDEventDriver);
 
 void VoodooI2CPrecisionTouchpadHIDEventDriver::enterPrecisionTouchpadMode() {
-    digitiser.input_mode->setValue(INPUT_MODE_TOUCHPAD);
+    if (version_major > CATALINA_MAJOR_VERSION) {
```

Comment:
Just a comment, where is `version_major` coming from? Is it from the `<libkern/version.h>` header?

Reply:
It is, https://github.com/acidanthera/MacKernelSDK/blob/master/Headers/libkern/version.h#L79